nixops/resources/ec2_common.py

```diff
@@ -19,7 +19,7 @@ def _retry(self, fun, **kwargs):
     def get_common_tags(self):
         tags = {'CharonNetworkUUID': self.depl.uuid,
                 'CharonMachineName': self.name,
-                'CharonStateFile': "{0}@{1}:{2}".format(getpass.getuser(), socket.gethostname(), self.depl._db.db_file)}
+                'CharonStateFile': "{0}@{1}:{2}".format(getpass.getuser(), socket.gethostname(), self.depl._state.db.db_file)}
```

Comment:
I'm getting this error

```
Traceback (most recent call last):
  File "/nix/store/qyqsfqjpksrvaa387yk4y5bgxnslfyx8-python2.7-nose-1.3.7/lib/python2.7/site-packages/nose/case.py", line 197, in runTest
    self.test(*self.arg)
  File "/nix/store/kwq9a1bjhc3w1672khipz85qrwpxj5lv-python2.7-parameterized-0.6.1/lib/python2.7/site-packages/parameterized/parameterized.py", line 280, in <lambda>
    nose_func = wraps(func)(lambda *args: func(*args))
  File "/home/srghma/projects/nixops/tests/functional/test_ec2_backups/__init__.py", line 30, in test_ec2_backups
    deployment.deploy()
  File "/home/srghma/projects/nixops/nixops/deployment.py", line 989, in deploy
    self.run_with_notify('deploy', lambda: self._deploy(**kwargs))
  File "/home/srghma/projects/nixops/nixops/deployment.py", line 978, in run_with_notify
    f()
  File "/home/srghma/projects/nixops/nixops/deployment.py", line 989, in <lambda>
    self.run_with_notify('deploy', lambda: self._deploy(**kwargs))
  File "/home/srghma/projects/nixops/nixops/deployment.py", line 922, in _deploy
    nixops.parallel.run_tasks(nr_workers=-1, tasks=self.active_resources.itervalues(), worker_fun=worker)
  File "/home/srghma/projects/nixops/nixops/parallel.py", line 44, in thread_fun
    result_queue.put((worker_fun(t), None, t.name))
  File "/home/srghma/projects/nixops/nixops/deployment.py", line 895, in worker
    r.create(self.definitions[r.name], check=check, allow_reboot=allow_reboot, allow_recreate=allow_recreate)
  File "/home/srghma/projects/nixops/nixops/backends/ec2.py", line 1012, in create
    self.update_tags(self.vm_id, user_tags=common_tags, check=check)
  File "/home/srghma/projects/nixops/nixops/resources/ec2_common.py", line 45, in update_tags
    self.update_tags_using(updater, user_tags=user_tags, check=check)
  File "/home/srghma/projects/nixops/nixops/resources/ec2_common.py", line 33, in update_tags_using
    tags.update(self.get_common_tags())
  File "/home/srghma/projects/nixops/nixops/resources/ec2_common.py", line 22, in get_common_tags
    'CharonStateFile': "{0}@{1}:{2}".format(getpass.getuser(), socket.gethostname(), self.depl._state.db.db_file)}
AttributeError: TransactionalJsonFile instance has no attribute 'db_file'

```

So I will rename `self._db_file` for json state to `self.db_file`